docs/user/manifests/kubeapps-local-dev-dex-values.yaml

```diff
@@ -19,6 +19,7 @@ config:
   # one.
   - id: default
     redirectURIs:
+    - 'http://localhost/oauth2/callback'
```

Comment:
Yes, they are for the CI context. Note that no ingress (no https) is being used during the tests. I initially thought of adding it via scripting in the CI config file. However, since it is also useful when setting up the dev environment and running the tests there, I opted to modify this file directly.